src/storage/buffer/buffer_obj.cpp

```diff
@@ -97,14 +99,16 @@ void BufferObj::UnloadInner() {
 
 bool BufferObj::Free() {
     std::unique_lock<std::shared_mutex> w_locker(rw_locker_);
+    auto status = status_;
     switch (status_) {
         case BufferStatus::kFreed:
         case BufferStatus::kLoaded: {
             // loaded again after free, do nothing.
             // Or has been freed in fronter of the queue.
             return false;
         }
-        case BufferStatus::kUnloaded: {
+        case BufferStatus::kUnloaded:
+        case BufferStatus::kClean: {
```

Comment:
Free a CLEAN status object. Shall we free the memory and remove it from buffer manager?